src/palace/manager/search/service.py

```diff
@@ -336,20 +340,20 @@ def write_pointer_set(self, revision: SearchSchemaRevision) -> None:
 
     def read_pointer(self) -> str | None:
         try:
-            result: dict = self._client.indices.get_alias(name=self.read_pointer_name())
+            result = self._client.indices.get_alias(name=self.read_pointer_name())
             for name in result.keys():
                 if name.startswith(f"{self.base_revision_name}-"):
-                    return name
+                    return name  # type: ignore[no-any-return]
```

Comment:
Hmmm. It looks like `name` should be a `str` here.

Reply:
From what I could see client is untyped so `self._client.indices.get_alias` ends up giving us an `Any` type, which means `name` ends up being `Any`. I type hinted the returned dict, so we get a better type then `Any`.